### PR TITLE
Change Fedora latest to auto-detection (for fedora 25)

### PR DESCRIPTION
**Pull request by jedigo**
Add Fedora support for -p
Add Fedora nonfree codec installation

Hi,
This should be the last update for a while:

- Auto detect the latest Fedora version (similar to Ubuntu),
- Install extra packages
- Install non-free codecs (MP3, MPEG2 etc) with the customizations enabled
- Allow for installing the development/rolling branch

This is pretty much everything I can think of to bring this branch up to parity with the Ubuntu installation.

### CODE REVIEW

**Comment by reynhout** [comment]
Wow, very nice. Thanks, I will take a closer look this week.

File: dist/chrx-install-chroot [hunk 1]
```diff
@@ -350,7 +415,6 @@ install_google_chrome()
 
   apt_get_update_if_needed
   echo_title "Installing Google Chrome."
-  apt_get_install google-chrome-stable
```
Comment (reynhout) [inline dist/chrx-install-chroot]: Still need this line for Ubuntu installs

File: dist/chrx-install-chroot [hunk 2]
```diff
@@ -361,9 +425,37 @@ fedora_install_google_chrome()
...
+
+  dnf ${VERBOSITY_DNF} -y install gstreamer1 gstreamer1-plugins-base gstreamer1-plugins-base-tools gstreamer1-plugins-good gstreamer1-plugins-good-extras gstreamer1-plugins-bad-free gstreamer1-plugins-bad-free-extras gstreamer1-plugins-bad-freeworld gstreamer1-plugins-ugly gstreamer1-libav
+
+  dnf ${VERBOSITY_DNF} -y install faad2 faac libdca compat-libstdc++-33 compat-libstdc++-296 xine-lib-extras-freeworld
+}
+
 install_desktop_support()
 {
   [ ! -x /usr/bin/X ] && return ## only if we have X
+  [ ! "${CHRX_CUSTOMIZATION_ENABLED}" ] && return
```
Comment (reynhout) [inline dist/chrx-install-chroot]: Thanks, missed that.

**Comment by jedigo** [comment]
Thanks, I guess I must have cut a line I didn't intend to there.

I didn't get time to look at this during the week, but it looks as though you sorted it out already.

**Comment by reynhout** [comment]
@jedigo Yep, I had a couple of other changes to make, so I went ahead and merged it all.

The chrx logs show about 100 completed Fedora installs so far. :)

**Pull request merged**